Extension.php

```diff
@@ -11,10 +12,40 @@ public function initialize()
     {
         $this->addCss('assets/extension.css');
         $this->addJavascript('assets/start.js', true);
```

Comment:
Just thinking about this a bit more… these are better in the controller's before, but we're deliberately mixing up internal route methods with a controller class for demonstration. So maybe do both, just add comments (and spearate file names) to point that out